boefjes/boefjes/plugins/kat_maxmind_geoip/main.py

```diff
@@ -25,8 +26,13 @@
 
 def run(boefje_meta: BoefjeMeta) -> list[tuple[set, bytes | str]]:
     input_ = boefje_meta.arguments["input"]
+    ip = input_["address"]
     hash_algorithm = getenv("HASHFUNC", HASHFUNC)
 
+    # if the address is private, we do not need a Location
+    if not ip_address(ip).is_global:
+        return [(set(), json.dumps("IP address is private, no location possible"))]
```

Comment:
This is indeed the correct solution. It will produce a warning in the raw file, but won't be consumed by the normalizer